src/main/java/lits/jp/hotel/management/places/response.txt

```diff
@@ -0,0 +1 @@
f=\"https://maps.google.com/maps/contrib/107115135069482925888/photos\"\u003eJakub Kaznowski\u003c/a\u003e"],"photo_reference":"CmRaAAAAi-BSeV5tlOHZAV7TScKF1XHKFq_wQ2zQvjdf4Jx99IgGGrVBb2vaQA2oCRzHOFePVb4__k0ziRbFra-b-_m66OQrB_mk9e54fPHvX-tLpG-fY0FrW7nIwz9838kj_gGMEhBKCuzfPaRG0T0tJ_WfgFs0GhSVtlw-in0iRGa-sGFuMj5Bj8GPRA","width":2988 } ],"place_id":"ChIJN_jzJGzdOkcRALejoB3A9S8","plus_code":{"compound_code":"R2RJ+2V ����, �������� �������, ������","global_code":"8GX6R2RJ+2V"},"price_level":2,"rating":4.4,"reference":"ChIJN_jzJGzdOkcRALejoB3A9S8","scope":"GOOGLE","placeTypes":["bar","point_of_interest","establishment"],"user_ratings_total":769,"vicinity":"������ ����� ���������, 10, ����"}, {"geometry":{"location":{"lat":49.8405938,"lng":24.0319713 },"viewport":{"northeast":{"lat":49.8419050802915,"lng":24.0333412302915 },"southwest":{"lat":49.8392071197085,"lng":24.0306432697085 } } },"icon":"https://maps.gstatic.com/mapfiles/place_api/icons/bar-71.png","id":"889f5b2c9b117f7a9d13c1b40017c229065db031","name":"������� � ������","opening_hours":{"open_now":false },"photos":[ {"height":2328,"html_attributions":["\u003ca href=\"https://maps.google.com/maps/contrib/100820504604822736194/photos\"\u003eYurij Zagrebnoy\u003c/a\u003e"],"photo_reference":"CmRaAAAADTmiRKb9aW-mD4PhLF7Jv6JVgK2S9Q4UlAKIhlbeheY3HYvMYZuzYPx3gncoDD0JyhoSm06Mcli1Gs2nTNFnzcGL-8pdszCjLFvWw_LmouexDtrvxaSAOcacucfIBCJYEhDFW9M8oFYUpU0WIOZXtsalGhSR79c_8sApGNG7Ictnp5vYupk0Iw","width":4656 } ],"place_id":"ChIJLzw1P2zdOkcRfABcGLU6f4A","plus_code":{"compound_code":"R2RJ+6Q ����, �������� �������, ������","global_code":"8GX6R2RJ+6Q"},"price_level":2,"rating":4.6,"reference":"ChIJLzw1P2zdOkcRfABcGLU6f4A","scope":"GOOGLE","placeTypes":["bar","point_of_interest","establishment"],"user_ratings_total":82,"vicinity":"������ �������������, 7, ����"}, {"geometry":{"location":{"lat":49.841673,"lng":24.030309 },"viewport":{"northeast":{"lat":49.8430596802915,"lng":24.0318581302915 },"southwest":{"lat":49.8403617197085,"lng":24.02916016970849 } } },"icon":"https://maps.gstatic.com/mapfiles/place_api/icons/cafe-71.png","id":"8167e175335a35e920357b2480cea050baa5dc51","name":"���� ����","opening_hours":{"open_now":false },"photos":[ {"height":1251,"html_attributions":["\u003ca href=\"https://maps.google.com/maps/contrib/116420520608877579677/photos\"\u003e���� ����\u003c/a\u003e"],"photo_reference":"CmRaAAAA8gdV5Z15-jTvdA8ncv8ZcjX9z7wnV5q7hVFbvab8DUZ3e6FMjLmLxXN3bWgWgSRf4R8A5vYTszVEVQi-aoAfbyU1J6SPBYRbzcrB6kTma4-GZtVFoRzZvcQPI6h8cRMJEhDDK0sXPLq8GDAR5q0RURhhGhSjsQDvjc-vZ9-m19uGcN5Zi1WdHw","width":1875 } ],"place_id":"ChIJs--2vW3dOkcRQpB3AkElOg0","plus_code":{"compound_code":"R2RJ+M4 ����, �������� �������, ������","global_code":"8GX6R2RJ+M4"},"price_level":2,"rating":4.4,"reference":"ChIJs--2vW3dOkcRQpB3AkElOg0","scope":"GOOGLE","placeTypes":["cafe","night_club","bar","food","point_of_interest","establishment"],"user_ratings_total":432,"vicinity":"���'����� �������������, ����� �����, 29, ����"} ],"status":"OK"}
```

Comment:
Is this a configuration file or a generated response? If the latter, can it not be on the repo?

Reply:
It is generated response. there is a limit for real responses per day from Google.